try:
    passe

#     i = 1
#     while i < 10:
#         print(i)
#         i += 1
except Exception as e:
    print(f"There is an error: {e}")
